### ISSUE TITLE
In word when typing the non-breaking space symbol done by pressing ctrl + shift + space nvda does not announce the word that has finished being typedeeeeeeeeeeet

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#14902]
### Steps to reproduce:
Open a new document in word type the word Example Fernando (press ctrl + shift + space to enter the non-breaking space character), type Silva

### Actual behavior:
Currently nvda does not announce non-breaking space when pressing ctrl+shift+space. Currently, when you finish typing the first word Fernando and type ctrl + shift + space, nvda does not announce the word that has just been typed in the case of Fernando.

### Expected behavior:
I hope that when typing ctrl + shift + space nvda pronounces the space symbol without breaking When typing the first word and then pressing ctrl + shift + space I expect to hear the first word typed in the Fernando case. I was unable to paste the non-breaking space symbol into nvda dialogs as dictionary or symbol dictionary.

#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
alfa nvda.exe, NVDA alpha-28166,e77bde9a

#### Windows version:
10 19045.2913
#### Name and version of other software in use when reproducing the issue:
WINWORD.EXE, Microsoft Office 2016 16.0.4266.1001

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by Brian1Gaff** [comment]
Maybe I call this something else, as I've never heard of it.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message ----- 
From: "fernando-jose-silva" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Wednesday, May 03, 2023 10:17 PM
Subject: [nvaccess/nvda] In word when typing the non-breaking space symbol 
done by pressing ctrl + shift + space nvda does not announce the word that 
has finished being typedeeeeeeeeeeet (Issue #14902)
